packages/react-youtube/src/YouTube.tsx

```diff
@@ -240,13 +240,13 @@ class YouTube extends React.Component<YouTubeProps> {
     this.createPlayer();
   }
 
-  componentDidUpdate(prevProps: YouTubeProps) {
+  async componentDidUpdate(prevProps: YouTubeProps) {
     if (shouldUpdatePlayer(prevProps, this.props)) {
       this.updatePlayer();
     }
 
     if (shouldResetPlayer(prevProps, this.props)) {
-      this.resetPlayer();
+      await this.resetPlayer();
```

Comment:
This seems suspisious... If this where to reject it'd be an unhandled promise rejection because react doesn't do anything special with the return value of `componentDidUpdate()`

Reply:
This was already the case before since we changed to promises.

Reply:
Yea I guess we weren't handing this either way...